pysb/bng.py

```diff
@@ -924,6 +924,47 @@ def _is_bool_expr(e):
     return isinstance(e, Boolean) and not isinstance(e, sympy.AtomicExpr)
 
 
+# BNG's expression syntax supports multiplication with a boolean, e.g.
+# `2 * (x > 0)`, but sympy does not. Our overall strategy is to convert
```

Comment:
Wouldn't there be the same issue for other operands such as `/` or `pow`?

Reply:
Yeah, but our previous code here only ever fixed up `*` so I could consider this change just a fix for the new stricter sympy semantics. Adding support for the other operators should be a separate change. In practice I think the `*` version of this is the only one that's been seen "in the wild" in BNG models so far.